data/lavezzo2020suppression/lavezzo2020suppression-extraction.md

```diff
@@ -0,0 +1,238 @@
+                                                    biomarker="SARS-CoV-2",
+                                                    gene_target="RdRp",
+                                                    limit_of_quantification="unknown",
+                                                    limit_of_detection="unknown",
+                                                    unit="gc/mL",
+                                                    reference_event="confirmation date"),
+                             E_first_pos=dict(description=folded_str("SARS-CoV-2 RNA genome copy concentration calculated from evaluation of E gene, and the reference event is the first positive test day (confirmation date).\n"),
+                                              specimen="oropharyngeal_swab",
+                                              biomarker="SARS-CoV-2",
+                                              gene_target="E",
+                                              limit_of_quantification="unknown",
+                                              limit_of_detection="unknown",
+                                              unit="gc/mL",
+                                              reference_event="confirmation date"),
+                            RdRp_symptom=dict(description=folded_str("SARS-CoV-2 RNA genome copy concentration calculated from evaluation of RdRp gene, and the reference event is symptom onset.\n"),
+                                                specimen="oropharyngeal_swab",
+                                                biomarker="SARS-CoV-2",
+                                                gene_target="RdRp",
+                                                limit_of_quantification="unknown",
+                                                limit_of_detection="unknown",
+                                                unit="gc/mL",
+                                                reference_event="symptom onset"),
+                             E_symptom=dict(description=folded_str("SARS-CoV-2 RNA genome copy concentration calculated from evaluation of E gene, and the reference event is symptom onset.\n"),
+                                              specimen="oropharyngeal_swab",
+                                              biomarker="SARS-CoV-2",
+                                              gene_target="E",
+                                              limit_of_quantification="unknown",
+                                              limit_of_detection="unknown",
+                                              unit="gc/mL",
+                                              reference_event="symptom onset")),
+               participants=participant_list)
+
+```
+
+```python
+with open("lavezzo2020suppression.yaml","w") as outfile:
+    outfile.write("# yaml-language-server: $schema=../.schema.yaml\n")
+    yaml.dump(Lavezzo2020, outfile, default_style=None, default_flow_style=False, sort_keys=False)
```

Comment:
I rename the dataframe from "haisuproj" to "Lavezzo2020".

Reply:
Renamed